<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhoneDictionary</class>
 <widget class="QMainWindow" name="PhoneDictionary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhoneDictionary</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>21</y>
      <width>27</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mail">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>63</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_memo">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>110</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>memo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_update">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>471</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>mail</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>memo</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_name">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>21</y>
      <width>256</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_mail">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>63</y>
      <width>256</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_memo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>256</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_view">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>637</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionStatic"/>
    <addaction name="actionHeap"/>
    <addaction name="actionFile"/>
    <addaction name="actionDB"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStatic">
   <property name="text">
    <string>Static</string>
   </property>
  </action>
  <action name="actionHeap">
   <property name="text">
    <string>Heap</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionDB">
   <property name="text">
    <string>DB</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
